.travis.yml

```diff
@@ -18,6 +18,7 @@ before_script:
   - mkdir -p "$R_LIBS_USER"
   - Rscript -e 'if (length(find.package("devtools", quiet = TRUE)) == 0L) { install.packages("devtools", repos = "http://cran.rstudio.com") }'
   - Rscript -e 'devtools::update_packages("devtools", repos = "http://cran.rstudio.com")'
+  - Rscript -e 'if (!require("roxygen")) devtools::install_github("klutometis/roxygen")'
```

Comment:
I think so, to avoid this build error: https://travis-ci.org/robertzk/cachemeifyoucan/builds/118571637